ports/anyrpc/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "anyrpc",
-  "version-string": "2020-01-13",
-  "port-version": 2,
+  "version-date": "2021-08-24",
+  "port-version": 1,
```

```suggestion
  "port-version": 0,
```
Since this is version update, it should go back to 0. Or remove this field.